app/build.gradle.kts

```diff
@@ -94,6 +95,10 @@ dependencies {
     implementation("androidx.preference:preference:1.2.1")
     implementation("com.google.android.material:material:1.12.0")
     implementation("com.github.yalantis:ucrop:2.2.9")
+    implementation("androidx.tracing:tracing:1.2.0")
```

Comment:
Is this dependency necessary? Given only new tests are being added, it seems weird to me to add a new runtime dependency.

Reply:
You are probably right this dependency might not be necessary. While trying to integrate fastlane screengrab (unsuccessfully) within these tests I had to add some more dependencies and probably did not remove them.

Reply:
I can see that the tests fail if you remove this, it seems to be a dependency of Espresso.

However, given it is an Espresso test dependency and not a runtime dependency, it seems to me this should be an `androidTestImplementation` instead of `implementation` so we don't ship it with the actual APKs users will install as it's not needed there.

So that's what I tried... and it didn't work. And then I researched more. And... it seems to be an AGP bug. Sigh: https://github.com/android/android-test/issues/1755#issuecomment-1523813685

I decided to ask yet again for a way to track this issue as the previous request for that got no answer. It's definitely an ugly workaround, but it seems the only option currently: https://github.com/android/android-test/issues/1755#issuecomment-2143360036